### DISCUSSION TITLE
[Feature Request] outline-width utilities?

### DISCUSSION THREAD

**Discussion by abdullahtariq1171** [tailwindlabs/tailwindcss#1142] [Ideas]
Currently there is no utility to set width of outline, How about we add `outline-width` utilities:

```
.outline-thin
.outline-medium
.outline-thick

```

We can also allow to add more variants through theme extension.

[Mozilla docs reference ](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-width)

**Comment by royvanv** [comment]
I would propose using the same widths as the border width classes, because the variants you listed differ per browser.

```css
.outline-2
.outline-4
.outline-8
```

Or perhaps extend `shadow-outline`?

**Comment by hacknug** [comment]
Already possible using [core's `outline` plugin](https://tailwindcss.com/docs/outline/) since it uses the shorthand property. Probably better to have individual plugins to allow composition so I'll look into releasing a plugin for it 👍
